MyFirstApplication/Models/AppSettings.cs

```diff
@@ -5,4 +5,16 @@ public class AppSettings
         public string? BaseUri { get; set; }
         public int PageSize { get; set; }
     }
+
+    public class Authentication
+    {
+        public Google? GoogleSettings { get; set; }
+    }
```

Comment:
You probably don't need this extra class. You can define directly in AppSettings and bind it instead of reading as hardcode strings. Also, `GoogleAuthSettings` was a better name.

```suggestion
public class AppSettings
{
    public string? BaseUri { get; set; }
    public int PageSize { get; set; }
    public Google GoogleSettings { get; set; }
}
```